source/MCR20Drv.c

```diff
@@ -643,6 +645,7 @@ void MCR20Drv_RESET
 void
 )
 {
+  #if !defined(TARGET_KW24D)
```

Comment:
Hi Mac, Can we try running the test with out this commented out for KW24D. I am thinking the issue seen was related to the low-power code and not resetting the modem.